### ISSUE TITLE
Question: Possible to remove prefixes from variables?

### ISSUE THREAD

**Issue by josdemmers** [JosefPihrt/Roslynator#826]
Multiple C# projects that I work on use prefixes for variables.
For example the a-prefix for arguments/parameters.
l-prefix for local variables.

I was wondering if I can refactor this using Roslynator?
e.g. I have the following method:
`public Object ConvertBack(Object aValue, Type aTargetType, Object aParameter, CultureInfo aCulture)`
After refactoring that should become:
`public Object ConvertBack(Object value, Type targetType, Object parameter, CultureInfo culture)`

So basically from aUpperCamelCase to lowerCamelCase

**Comment by JosefPihrt** [comment]
I have a plan to add this kind of functionality to Roslynator CLI.

In this concrete case the command would do a batch rename in the entire solution and it would probably look like this:
```
roslynator symbol rename --match "symbol.Kind == SymbolKind.Parameter && Regex.Match(symbol.Name, ""^a[A-Z]"")" ^
  --replacement "symbol.Name[1].ToString().ToLower() + symbol.Name.Substring(2)"
```

The work in progress is here: https://github.com/JosefPihrt/Roslynator/tree/cli-rename

**Comment by josdemmers** [comment]
Thank you, that will be very useful.
While searching for this a came also across this Roslyn script to handle local variables and parameters:
https://stackoverflow.com/a/40660819
